blocks/form/lib/forms-common.js

```diff
@@ -398,7 +398,10 @@ function enableValidation(form) {
 
 async function handleSubmit(e, form, captcha, submitHandler) {
   e.preventDefault();
+  const hiddenFields = form.querySelectorAll('[data-visible="false"]');
+  hiddenFields.forEach(((field) => { field.disabled = true; })); // disabled fields are skipped for validation
```

Comment:
please check if this data need to be sent to backend or not? If its disabled I doubt data will not be sent to server

Reply:
Data is getting submitted